src/System.Diagnostics.PerformanceCounter/tests/PerformanceCounterCategoryTests.cs

```diff
@@ -0,0 +1,368 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
BTW I noticed lingering use of CAS (code access security) in performancecountercategory.cs:
```
            PerformanceCounterPermission permission = new PerformanceCounterPermission(PerformanceCounterPermissionAccess.Read, machineName, categoryName);
            permission.Demand();
```

any stuff that looks like this can be removed.

Reply:
You can also remove any of these
`RuntimeHelpers.PrepareConstrainedRegions();`

Reply:
Can I also remove RegistryPermission, EnvironmentPermission and FileIOPermission, etc?

Reply:
yes, anything in S.S.Permissions -- remove all the `using System.Security.Permissions;` and you will find them